Diffusion/Common/Pipeline/Pipeline.swift

```diff
@@ -81,6 +81,7 @@ class Pipeline {
         config.guidanceScale = guidanceScale
         config.disableSafety = disableSafety
         config.schedulerType = scheduler
+        config.useDenoisedIntermediates = true
```

Comment:
@pcuenca With this now merged https://github.com/apple/ml-stable-diffusion/pull/210 there's just a new setting required in the config to use denoised latents (noisy by default). Not sure we need one here as well, maybe in the advanced settings?

Reply:
In my opinion, I'd just use the denoised ones, unless you think there's a case for supporting both :)

Reply:
Agreed, denoised is preferred, and it demos the new config option 👍